server/liveblog/syndication/syndication.py

```diff
@@ -94,6 +97,12 @@ def send_syndication_post(self, post, action='created'):
             logger.warning('Not sending post "{}": syndicated content.'.format(post['_id']))
             return
 
+        items = extract_post_items_data(post)
+        for item in items:
+            if item['group_type'] == 'freetype' and item['item_type'] in SYNDICATION_EXCLUDED_ITEMS:
+                logger.warning('Not sending post "{}": syndicated content contains excluded items.'.format(post['_id']))
```

Comment:
This is an expected case, so logging with warning level is not appropriate, I think. Should be logger.debug